docs/pages/ESSupport.md

```diff
@@ -8,7 +8,7 @@ UI5 Tooling offers general support for `ES2022` ECMAScript features. While a `ui
 | v3.0+               | ECMAScript 2022              |      |
 | v2.0+               | ECMAScript 2009/ES5          | Note that code up to ECMAScript 2020 can be parsed, however required code analysis might not work correctly for specific language features |
 
-In section [Language Features with Limitations](#language-features-with-limitations) all limitations grouped by the kind of ECMAScript artefact are described. In section [Code Analyzing](#code-analyzing) there is a more detailed explaination what kind of code analysis is done by the UI5 Tooling.
+The following section describes all limitations grouped by the kind of ECMAScript language feature. To get more insights into the code analysing executed by UI5 Tooling check out [Code Analysis](./CodeAnalysis.md).
 
 ## Language Features with Limitations
```

Comment:
@ecker : shouldn't the title of this section rather be "Language Features with _Restrictions_"?

Reply:
Yes, please avoid usage of "limitations"; use "restrictions" (or similar in text) instead.

Reply:
done